src/include/executor/executor.h

```diff
@@ -475,7 +475,7 @@ extern Datum ExecMakeFunctionResultSet(SetExprState *fcache,
 									   ExprDoneCond *isDone);
 extern void ExecSquelchFunctionResultSet(SetExprState *fcache,
 										 ExprContext *econtext,
-										 MemoryContext *argcontext);
```

Comment:
Why we change the function signature and do not fix incorrect function call? Why do we break ABI tests? If the reason is signinficant why not to tell about it in patch description?